addons/account_peppol/models/account_move_send.py

```diff
@@ -139,18 +146,11 @@ def _call_web_service_after_invoice_pdf_render(self, invoices_data):
         params = {'documents': []}
         invoices_data_peppol = {}
         for invoice, invoice_data in invoices_data.items():
-            if 'peppol' in invoice_data['sending_methods'] and self._is_applicable_to_move('peppol', invoice):
-                if invoice_data.get('ubl_cii_xml_attachment_values'):
-                    xml_file = invoice_data['ubl_cii_xml_attachment_values']['raw']
-                    filename = invoice_data['ubl_cii_xml_attachment_values']['name']
-                elif invoice.ubl_cii_xml_id and invoice.peppol_move_state not in ('processing', 'done'):
-                    xml_file = invoice.ubl_cii_xml_id.raw
-                    filename = invoice.ubl_cii_xml_id.name
-                else:
-                    invoice.peppol_move_state = 'skipped'
-                    continue
-
-                partner = invoice.partner_id.commercial_partner_id.with_company(invoice.company_id)
+            partner = invoice.partner_id.commercial_partner_id.with_company(invoice.company_id)
+            if 'peppol' in invoice_data['sending_methods']:
+                invoice_edi_format = invoice_data['invoice_edi_format'] or partner._get_peppol_edi_format()
```

Comment:
Can you confirm at this point ` or partner._get_peppol_edi_format()` is "useless" because of `_get_default_invoice_edi_format` that is supposed to have already compute the correct `invoice_edi_format` ? :thinking: 

If yes, then `_get_peppol_edi_format` should be used in `_get_default_invoice_edi_format`, correct?